src/com/google/enterprise/adaptor/MetadataTransform.java

```diff
@@ -25,34 +27,57 @@
  */
 public interface MetadataTransform {
 
-  /** Keys for data stored in the {@code Map} of {@code params}. */
+  /** The key for the document's unique ID supplied in the {@code params}. */
   public static final String KEY_DOC_ID = "DocId";
+
+  /** The key for the document's display URL supplied in the {@code params}. */
   public static final String KEY_DISPLAY_URL = "Display-URL";
+
+  /** The key for the document's content type supplied in the {@code params}. */
   public static final String KEY_CONTENT_TYPE = "Content-Type";
+
+  /** The key for the document's crawl once flag supplied in the {@code params}.
+   */
   public static final String KEY_CRAWL_ONCE = "Crawl-Once";
+
+  /** The key for the document's lock flag supplied in the {@code params}. */
   public static final String KEY_LOCK = "Lock";
+
+  /** The key for the document's last modified timestamp supplied in the
```

Comment:
This is a style guide violation. Implementation comments can be formatted like this, but not javadoc.

https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting